### DISCUSSION TITLE
Filling item data directly in table view

### DISCUSSION THREAD

**Discussion by maks2199** [directus/directus#18687] [Feature Requests]
### Summary

Is it possible to add function to fill data when creating new item in collection not using card view, but staying in table view, like in Microsoft Access? 

### Motivation

It is vital for some users, especially oldscool one, to write data without opening card view.

### Detailed Design

![image](https://github.com/directus/directus/assets/55157621/d2366ae6-23d3-4b4a-95bb-caa29504fda1)
![image](https://github.com/directus/directus/assets/55157621/a5f6165f-2419-403b-9c45-e52ac65b4e83)

### Must Have:
- opportunity to write field data directly in table, without opening card view

### Drawbacks

It sounds like not the phylossophy of database management system. But Directus declares himself as user-friendly interface for db management, so it shuold have different ways to input data and satisfy as many users as possible

### Alternatives

Using external interfaces and Directus API ti implement this

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
This is a duplicate of the existing discussion https://github.com/directus/directus/discussions/17937. Make sure to give that one an upvote!
